src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -116,9 +116,12 @@ def resolve_data(
                     else:
                         missing_inputs.append(name)
                         continue
-                    dependency_nodes_outputs[value.value] = (
-                        {value.property: dependency_input} if value.property else dependency_input
-                    )
+                    if value.property:
```

Comment:
please add more PR description on this change.

Reply:
update PR description